rover-apps/arm_2021/CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
 add_executable(arm_2021)
-target_sources(arm_2021 PRIVATE src/main.cpp)
-target_include_directories(arm_2021 PUBLIC include)
+target_sources(arm_2021 PRIVATE ../common/src/main.cpp)
+target_include_directories(arm_2021 PUBLIC include ../common/include)
```

Comment:
sorta gross to have relative paths in cmake like this. is there anyway to have some type of target in the common folder and link against that target here or smth? not sure... if u cant figure out somethign quick here, then could just leave it as a "later" cleanup

Reply:
I'm not sure if you can have main.cpp in a library, gonna change it to `${ROVER_APPS_DIR}/common` instead